src/ray/gcs/gcs_server/gcs_server_main.cc

```diff
@@ -66,6 +66,7 @@ int main(int argc, char *argv[]) {
 
   const ray::stats::TagsType global_tags = {
       {ray::stats::ComponentKey, "gcs_server"},
+      {ray::stats::WorkerIdKey, ""},
```

Comment:
Should we not include worker id for gcs & raylet (is there any reason to add an empty id there)?

Reply:
In principle I don't think it's needed, but our current metrics infra seems to assume metrics will be emitted with the same set of tags from different component. If different tag labels are used, the metric collector raises an error (metric mismatches auto generated views). This doesn't seem easy to fix without revamping view auto-generation.